api/app/Repositories/TestRepository.php

```diff
@@ -0,0 +1,14 @@
+<?php namespace App\Repositories;
+
+class TestRepository extends BaseRepository
+{
+    /**
+     * Model name.
+     *
+     * @return string
+     */
+    protected function model()
+    {
+        return 'App\Models\TestEntity';
```

Comment:
Would it be better to have the model defined as a static property? I think this would be better performance, but would it be a good design decision? I'm undecided.